test/framework/dump.go

```diff
@@ -182,20 +182,20 @@ func (f *CommonFramework) DumpLogsForPodsWithLabelsInNamespace(ctx context.Conte
 
 	var result error
 	for _, pod := range pods.Items {
-		if err := f.DumpLogsForPodInNamespace(ctx, k8sClient, namespace, pod.Name); err != nil {
+		if err := f.DumpLogsForPodInNamespace(ctx, k8sClient, namespace, pod.Name, ""); err != nil {
 			result = multierror.Append(result, err)
 		}
 	}
 	return result
 }
 
 // DumpLogsForPodInNamespace prints the logs of the pod with the given namespace and name.
-func (f *CommonFramework) DumpLogsForPodInNamespace(ctx context.Context, k8sClient kubernetes.Interface, namespace, name string) error {
+func (f *CommonFramework) DumpLogsForPodInNamespace(ctx context.Context, k8sClient kubernetes.Interface, namespace, name, containerName string) error {
```

Comment:
Nit: I would vote to allow passing the whole options instead of the containerName only

```suggestion
func (f *CommonFramework) DumpLogsForPodInNamespace(ctx context.Context, k8sClient kubernetes.Interface, namespace, name string, options *corev1.PodLogOptions) error {
```